docs/source/index.md

```diff
@@ -1,9 +1,45 @@
-# Jupyter Project Documentation
+# Project Jupyter Documentation
 
-**Welcome to the Jupyter Project documentation**. This website acts as "meta" documentation for the Jupyter ecosystem.
-It has a collection of resources to navigate the tools and communities in this ecosystem, and to help you get started.
+Welcome to the Project Jupyter documentation site. As you'll read below,
+Jupyter is a large umbrella project that covers many different software
+offerings and tools, including the popular **JuyterLab** and **Jupyter**
+**Notebook** web-based notebook authoring and editing applications. This
+site acts as (meta) documentation for the entire Jupyter project.
 
-## Start Here
+## What is Jupyter?
+
+The name "Jupyter" is an ambiguous, overloaded term that can mean several
+different things in different contexts, so let's start by clarifying a few
+common, important pieces of the Jupyter ecosystem:
+
+- Project Jupyter: The umbrella project for interactive computational notebooks
+  and related interactive computing tools.
+- Jupyter Notebook: This is the name of the popular, lightweight web-based
+  notebook authoring application. It is also commonly used to refer to a saved
+  notebook document (.ipynb) file (for example "I made a Jupyter notebook with
+  my analysis of the data we captured in the lab").
+- JupyterLab: A modern, feature-rich web-based notebook editing application and
+  computational environment. Offers more features and functionality than
+  the light-weight experience provided by the Jupyter Notebook application.
+
+You can read more about other tools and products in the Jupyter ecosystem
+in the glossary, or further down on this page (Like JupyterLite, Jupyter
+Desktop and Jupyter Server). They all center around the goal of providing
+tools for computational notebooks and interactive computing.
+
+## Where do I start?
+
```

Comment:
Maybe a "Who is this for section"

"Data scientist doing interactive analysis -> just start using jupyter with `jupyter notebook`"
"Existing Jupyter users who wants to build their first extension -> uhmmm... base guide to extending jupyter"
"want to contribute or file a bug against jupyter -> contributing guide".

Where to start is too broad a question to answer.